index.html

```diff
@@ -16,7 +16,7 @@
 					<a href="#" class="hover_effect">George M'sapenda</a>
 				</div>
 				<button class="btn">
-					<img src="images/Menu_icon.png" alt="menu bar icon">
+					<img src="/images/Icon - Menu.png" alt="menu">
 				</button> 
 				<ul id="desktop_nav">
 					<li><a href="#" class="hover_effect">Portifolio</a></li>
```

Comment:
[OPTIONAL]
- [ ] You may consider adding some spacing between your nav links 
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200906053-744a6ba2-28d3-4b3e-8b3e-9158d989d540.png)
Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200906353-6a303681-8fde-45ff-90f2-6aae7552665d.png)